Snabble/Core/API/SnabbleAPI.swift

```diff
@@ -64,7 +64,19 @@ public extension Notification.Name {
     static var metadataLoaded = Notification.Name(rawValue: "io.snabble.metadataLoaded")
 }
 
+/**
+ * The main entry point for the SnabbleSDK.
+ *
+ * We recommend using `SnabbleAPI.setup(_:, completion:)` to initialize Snabble.
```

Comment:
Ich würde "we" in der Doku nicht benutzen, und "recommend" für den einzig möglichen Weg ist auch irgendwie komisch. Einfach `Use SnabbleAPI.setup(...) to initialize the SDK`?

Reply:
e9768f59b693a08542b53878a27bebd4b676c334